src/styles/header.scss

```diff
@@ -0,0 +1,74 @@
+ul,
+body {
+  margin: 0; /* Обнулення стилів */
+  padding: 0; /* Обнулення стилів */
+}
+
+body {
+  font-family: Roboto, serif;
```

Comment:
Consider adding a generic fallback font after 'Roboto', such as 'sans-serif', to ensure that the text is displayed correctly if the primary font fails to load.